docs/user/unsupported/mbstyle/spec.rst

```diff
@@ -0,0 +1,5702 @@
+Mapbox Style Specification
+==========================
+
+A Mapbox style is a document that defines the visual appearance of a map: what data to draw, the order to draw it in, and how to style the data when drawing it. A style document is a `JSON <http://www.json.org/>`__ object with specific root level and nested properties. This specification defines and describes these properties.
+
+The intended audience of this quick reference includes:
+
+-  Advanced designers and cartographers who want to write styles by hand
+-  GeoTools developers using the mbstyle module
+-  Authors of software that generates or processes Mapbox styles.
+
+The `Mapbox Style Specification <https://www.mapbox.com/mapbox-gl-style-spec>`__ is generated from the BSD `Mapbox GL JS <https://github.com/mapbox/mapbox-gl-js>`__ github repository, reproduced here with details on this GeoTools implementation.
+
+- Feature support is provided for the `Mapbox GL JS <https://www.mapbox.com/mapbox-gl-js/api/>`__, `Open Layers <http://openlayers.org>`__ and the GeoTools mbstyle module.
+- Where appropriate examples have been changed to reference `GeoWebCache <http://geowebcache.org/>`__.
+
+
+Root Properties
+---------------
+
+Root level properties of a Mapbox style specify the map's layers, tile sources and other resources, and default values for the initial camera position when not specified elsewhere.
+
+::
+
+    {
+        "version": 8,
+        "name": "Mapbox Streets",
+        "sprite": "mapbox://sprites/mapbox/streets-v8",
+        "glyphs": "mapbox://fonts/mapbox/{fontstack}/{range}.pbf",
```

Comment:
Something similar (with the {}) should be fine, since when parsing the mapbox style we will have to parse all the fonts anyway, and can do URL substitution then.